whois-rpsl/src/test/java/net/ripe/db/whois/common/rpsl/RpslObjectTest.java

```diff
@@ -386,20 +386,22 @@ public void compareRpslObjects_single() {
         assertThat(RpslObject.parse("mntner: \t  mnt  \t \n \t     a"), is(RpslObject.parse("mntner: mnt\n\ta")));
         assertThat(RpslObject.parse("mntner: \t  one \t two \t \n \t     a"), is(RpslObject.parse("mntner: one two\n\ta")));
         assertThat(RpslObject.parse("mntner:one \t two \t \n \t     a"), is(RpslObject.parse("mntner: one two\n\ta")));
-        assertThat(RpslObject.parse("mntner: mnt # comment"), is(RpslObject.parse("mntner: mnt")));
+        assertThat(RpslObject.parse("mntner: mnt # comment"), is(RpslObject.parse("mntner: mnt # comment")));
         assertThat(RpslObject.parse("mntner: mnt # comment"), is(RpslObject.parse("mntner: mnt # comment")));
         assertThat(RpslObject.parse("mntner: mnt# comment"), is(RpslObject.parse("mntner: mnt # comment")));
         assertThat(RpslObject.parse("mntner: mnt # com  ment"), is(RpslObject.parse("mntner: mnt # com ment")));
         assertThat(RpslObject.parse("mntner: mnt # com  ment"), is(RpslObject.parse("mntner: mnt #com ment")));
+/*
```

Comment:
Change commented test to `is(not())` ?

Reply:
I commented it because I am not sure if tests is correct. How come   "mntner: mnt two three four" should be equal to "mntner: mnt # one\n two\n+ three\n\tfour". I think this test scenario is wrong and should be deleted

Reply:
That test *was* correct:
`        assertThat(RpslObject.parse("mntner: mnt two three four"), is(RpslObject.parse("mntner: mnt # one\n two\n+ three\n\tfour")));`
The value should be the same regardless of formatting. I suggest just removing the `# one` comment as you're now including this in the equals check, and just test the attribute value comparison.
Or, test the value *and* comment are the same, regardless of formatting 
```
assertThat(RpslObject.parse("mntner: mnt two three four # one"), is(RpslObject.parse("mntner: mnt # one\n two\n+ three\n\tfour # one")));
```

Reply:
dded and chenged